README.md

```diff
@@ -60,6 +62,10 @@ Alternatively, you can create the S3 bucket and IAM resources manually. Here are
 
 ### Set up the host
 
+If you are using GCP, please follow the [GCP Setup](./docs/GCP_SETUP.md) instructions before you continue.
```

Comment:
The README would benefit from a bit of refactoring so someone trying to set up GCP doesn't have to flip back and forth between this and `GCP_SETUP.md`. Would you mind merging those instructions directly in here?